modules/ansible/templates-ansible/configure-os-for-sap/playbook-configure-os-for-sap.yml.tftpl

```diff
@@ -15,8 +15,7 @@
     sap_domain: ${sap_domain}
 
   roles:
-  - { role: ibm.power_linux_sap.powervs_prepare_rhel_sap, when: sap_solution is defined and ansible_distribution is match("RedHat*") }
-  - { role: ibm.power_linux_sap.powervs_prepare_sles_sap, when: sap_solution is defined and ansible_distribution is match("SLES*") }
+   - { role: ibm.power_linux_sap.powervs_configure_os_for_sap, when: sap_solution is defined, ignore_errors: true }
```

Comment:
ignore_errors: this should be removed. Playbook and terraform should fail if any of the tasks fail